defaults/main.yml

```diff
@@ -634,3 +632,13 @@ swift_accesscheck_file: "index.html"
 swift_accesscheck_user_name: "accesscheck"
 swift_operator_role: "swiftoperator"
 swift_service_project_name: "service"
+
+# Instruct the swift service checks to be ldap aware.
+swift_service_in_ldap: false
+
+# Holland venv enabled
+holland_venv_enabled: true
+holland_venv_bin: "/openstack/venvs/holland-{{ rpc_release }}/bin"
+
+# Set the openstack-release to testing by default
```

Comment:
I kinda agree on the failure by default, and have the tests/rpc-maas-overrides.yml to testing

We could even have a

```
assert:
  that:
    - "openstack_release is defined"
```